src/nnsight/models/UnifiedTransformer.py

```diff
@@ -28,13 +28,13 @@ class UnifiedTransformer(LanguageModel):
     """
 
     def __init__(
-        self, model: str, *args, processing: bool = True, **kwargs
+        self, model: str, *args, processing: bool = True, tokenizer: Optional[PreTrainedTokenizer] = None, **kwargs
     ) -> None:
         if processing:
-            hooked_model = HookedTransformer.from_pretrained(model, *args, **kwargs)
+            hooked_model = HookedTransformer.from_pretrained(model, *args, tokenizer=tokenizer, **kwargs)
         else:
             hooked_model = HookedTransformer.from_pretrained_no_processing(
-                model, *args, **kwargs
+                model, *args, tokenizer=tokenizer, **kwargs
             )
 
         self.tokenizer = hooked_model.tokenizer
```

Comment:
also why do you do that when it'll be done again in the LanguageModel init?

Reply:
Sorry what is the context behind this?

Reply:
in unifiedtransformer init, you set self.tokenizer and then call the super init which will run https://github.com/ndif-team/nnsight/blob/f64d8a963d72acd58f2eaae6b1a2fcbf4d29cb63/src/nnsight/models/LanguageModel.py#L141
which will set self.tokenizer again, right?